main/integration-tests/__test__/api-tests/workspace-type-candidates/get-workspace-type-candidates.test.js

```diff
@@ -14,18 +14,25 @@
  */
 
 const { runSetup } = require('../../../support/setup');
+const {
+  createDefaultServiceCatalogProduct,
+  deleteDefaultServiceCatalogProduct,
+} = require('../../../support/complex/default-integration-test-product');
 const errorCode = require('../../../support/utils/error-code');
 
 describe('Get workspace-type candidates scenarios', () => {
   let setup;
   let adminSession;
+  let productInfo;
 
   beforeAll(async () => {
     setup = await runSetup();
     adminSession = await setup.defaultAdminSession();
+    productInfo = await createDefaultServiceCatalogProduct(setup);
```

Comment:
Why is productInfo added if it's not used by the tests?

Reply:
Removed it. Thanks for the catch!